src/lib/isTaxID.js

```diff
@@ -1226,15 +1238,26 @@ export default function isTaxID(str, locale = 'en-US') {
     if (locale in sanitizeRegexes) {
       strcopy = strcopy.replace(sanitizeRegexes[locale], '');
     }
-    if (!taxIdFormat[locale].test(strcopy)) {
-      return false;
-    }
 
-    if (locale in taxIdCheck) {
-      return taxIdCheck[locale](strcopy);
+    // Check if locale has multiple TIN
```

Comment:
```suggestion
```
The code below is clear so it doesn't need a comment

Reply:
😅 will remove it.